### DISCUSSION TITLE
Log refresh button is too far away from the bottom of the log

### DISCUSSION THREAD

**Discussion by somenewaccountthen** [home-assistant/frontend#12613] [Dashboard] [answered]
**The refreshbutton in the log:** 
I keep having to scroll back up and down to refresh/see the end.  
I know it's all new and probably gonna change some more. 

but..
It would realy be helpfull if there also was a refresh at the bottom. Or.. have the refresh button always stay in view. Or have a second one next to the dropdown in the top right. (see epic graphic design attached)

Should be relatively simple.. when ppl press the new refresh next to dropdown just trigger same action as if the dropdown changed.

no? ¯\_(ツ)_/¯

![psbWBSx](https://user-images.githubusercontent.com/76587846/167200088-e4948f84-8fe7-4141-ae0a-b293cd819b8b.png)

**Comment by matthiasdebaat** [comment, marked as answer]
Yeah, lets make this consistent like we have on the `Updates` page. I've create this issue https://github.com/home-assistant/frontend/issues/12637
